packages/os/os.spec

```diff
@@ -417,7 +420,9 @@ install -p -m 0644 \
 
 %if %{with aws_platform}
 install -p -m 0644 %{S:9} %{buildroot}%{_cross_templatedir}
+install -p -m 0644 %{S:10} %{buildroot}%{_cross_templatedir}
 install -p -m 0644 %{S:117} %{buildroot}%{_cross_unitdir}
+install -p -m 0644 %{S:121} %{buildroot}%{_cross_unitdir}
 %endif
```

Comment:
I suppose I can get behind this implementation for `aws-ecs-1` as well, although it's somewhat redundant there since there's an agent-level configuration in that variant.

Reply:
I think we should just use the ECS feature for ECS variants. There will be an additional IMDS call in the agent even if the instance is in `InService` which could increase "readyness" time.